autogen.pl

```diff
@@ -1020,6 +1021,27 @@ sub patch_autotools_output {
     unlink("configure.patched");
 }
 
+sub in_tarball {
+    my $tarball = 0;
+    open(IN, "VERSION") || my_die "Can't open VERSION";
+    # Read all lines from the file
+    while (<IN>) {
+          my $line = $_;
+          my @fields = split(/=/,$line);
+          if ($fields[0] eq "repo_rev") {
+              if ($fields[1] ne "\n") {
+                  $tarball = 1;
+              }
+          } elsif ($fields[0] eq "tarball_version") {
+              if ($fields[1] ne "gitclone\n") {
```

Comment:
Are you checking for `tarball_version=gitclone` simply as defensive programming?  (I.e., it should always catch the `repo_rev=` check, but might as well put in the check for `tarball_version`, too?)

My $0.02: Might be worth putting in a comment here explaining why you're checking `tarball_version`.